doc/Caliptra.md

```diff
@@ -1004,11 +1004,12 @@ This section describes Caliptra error reporting and handling.
 
 *Table 13: Hardware and firmware error types*
 
-| | Fatal errors | Non-fatal errors |
-| :- | - | - |
-| Hardware | - ICCM, DCCM SRAM ECC<br> - Second watchdog (WD) timer expiry. The first timer expiry triggers an NMI to firmware to correct the issue and clear the interrupt status bit. If the WD expires again, then it is escalated to a FATAL error. | - Mailbox SRAM ECC (except initial firmware load)<br> - Mailbox incorrect protocol or commands. For example, incorrect access ordering or access without Lock. |
-| Firmware | - Boot-time firmware authentication failures<br> - Firmware triggered FATAL errors. Examples: ICCM or DCCM have misaligned access (potentially in the except subroutine); AHB access hangs, triggered through WD timer expiry; AHB access outside of the decoding range; and stack overflow errors. For more information, see the table below. | - Mailbox API failures<br> - First WD timer expiry<br> - Cryptography processing errors |
+*Table 14: HW/FW Error Types*
 
+| | Fatal Errors | Non-fatal Errors |
+| :- | - | - |
+| HW | <p>- ICCM, DCCM SRAM ECC</p><p>- Second WD timer triggers an NMI and a FATAL error is signaled to the SoC.</p> | <p>- Mailbox SRAM ECC (except initial FW load)</p><p>- Mailbox incorrect protocol/commands. E.g. Incorrect access ordering; access without Lock.</p> |
```

Comment:
Accidentally reverted some tech writer changes:

HW -> Hardware
Second WD --> The second watchdog (WD)
(except initial FW load) --> (except initial firmware load)
protocol/commands --> protocol or commands
E.g. Incorrect access ordering; access without Lock. --> For example, incorrect access ordering or access without Lock.